data.sql

```diff
@@ -1,5 +1,18 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animal(id, animalname, animal_date_of_birth,
```

Comment:
Great job @Hassaanjbaig-code 🤝

- [ ] Kindly make sure you are inserting values into the table that you initially created. You should be inserting values into the `ANIMAL` table instead of `animal`.
![animal](https://user-images.githubusercontent.com/63917725/220396595-f8d17ffb-31a4-4790-8e3e-3306d24df555.PNG)